src/utils.rs

```diff
@@ -54,7 +55,7 @@ pub fn from_relocatable_to_indexes(relocatable: Relocatable) -> (usize, usize) {
 ///equivalent to val modulo prime.
 pub fn to_field_element(num: BigInt, prime: BigInt) -> BigInt {
     let half_prime = prime.clone() / bigint!(2);
```

Comment:
Idk if this is out of scope for the PR, but the half prime could be calculated with a shift right and the addition could be calculated with a ref instead of cloning

Reply:
I think it was a bit out of scope but since it is very little change I made it

Reply:
Nice! Thanks!